tests/test_staging/test_staging_helpers.py

```diff
@@ -428,7 +547,6 @@ def pc_mapper_output(self) -> pd.DataFrame:
         df = pd.DataFrame(columns=columns, data=data)
         return df
 
-
     def get_todays_date(self) -> str:
         """Get the date in the format YYYY-MM-DD. Used for filenames."""
         today = date.today()
```

Comment:
ISSUES RELEVANT TO THE CODE OUTSIDE OF THIS PR
- config param unused in `load_validate_secondary_snapshot`